src/__tests__/test.ts

```diff
@@ -1,7 +1,32 @@
-import { myModule } from '..';
+import { getDirectionalEllipse } from '..';
 
-describe('test myModule', () => {
-  it('should return 42', () => {
-    expect(myModule()).toBe(42);
+describe('getDirectionalEllipse', () => {
+  it('should return an annotation', () => {
+    let result = getDirectionalEllipse(
+      {
+        x: [1, 2, 3, 4, 5, 6],
+        y: [1, 4, 2, 5, 3, 6],
+      },
+      3,
+    );
+
+    expect(result).toStrictEqual({
```

Comment:
Always very dangerous to compare real numbers. Please add a dev dependency https://www.npmjs.com/package/jest-matcher-deep-close-to and ut it rather than toStrictEqual